### PR TITLE
API alter user help link default translations

### PR DESCRIPTION
**Pull request by NightJar**
In order to be able to facilitate help text links that are both adaptive to their authenticator and translated,
a new API was added to translate this help link text on the server, and pass this through to the components via
the 'available method' schema. Once the method component takes over (is rendered within the MFA container) translations
again revert to javascript provided context.

Consistency was also created in how these help links were looked up from config.

In relation to https://github.com/silverstripe/silverstripe-mfa/issues/142

### CODE REVIEW

**Comment by robbieaverill** [comment]
Needs a rebase

File: client/lang/src/en.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "MultiFactorAuthentication.FIND_OUT_MORE": "Find out more",
+  "MultiFactorAuthentication.FIND_OUT_MORE": "How multi-factor authentication works",
```
Comment (robbieaverill) [inline client/lang/src/en.json]: May be worth changing the key too?

File: client/src/components/BackupCodes/Register.js [hunk 2]
```diff
@@ -114,13 +114,15 @@ class Register extends Component {
...
-        >
-          {i18n._t('MFARegister.HELP', 'Find out more')}
-        </a>
+        {supportLink &&
+          <a
+            href={supportLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {i18n._t('MFARegister.RECOVERY_HELP', 'Learn more about recovery codes.')}
```
Comment (robbieaverill) [inline client/src/components/BackupCodes/Register.js]: Do you want to use `method.supportText` here? I see you've added it to the PHP class

File: client/src/components/BackupCodes/Verify.js [hunk 3]
```diff
@@ -86,13 +86,24 @@ class Verify extends Component {
...
           'Use one of the recovery codes you received'
         )}
+        &nbsp;
+        {method && method.supportLink &&
+          <a
+            href={method.supportLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {i18n._t('MFARegister.RECOVERY_HELP', 'How to use recovery codes.')}
```
Comment (robbieaverill) [inline client/src/components/BackupCodes/Verify.js]: This is different - is that by design? It's in its own component so it's fine if you want to leave it there, just asking

File: client/lang/src/en.json (hunk 1 above)
Comment (robbieaverill) [reply client/lang/src/en.json]: Missed this one

File: client/src/components/BackupCodes/Verify.js (hunk 3 above)
Comment (NightJar) [reply client/src/components/BackupCodes/Verify.js]: Yeah, it's by design :)

**Pull request merged**